test/browser.js

```diff
@@ -8,6 +8,14 @@ import withPage from './helpers/with-page.js';
 const test = ava.serial;
 const pBody = util.promisify(body);
 
+const kyScript = {
+	type: 'module',
+	content: `
+		import ky from './index.js';
+		window.ky = ky;
```

Comment:
Any ideas why this doesn't work? I'm still getting:

```
  browser › headers are preserved when input is a Request and there are searchParams in the options

  Rejected promise returned by test. Reason:

  Error {
    message: `Evaluation failed: TypeError: window.ky is not a function␊
        at __puppeteer_evaluation_script__:6:17`,
  }

  › __puppeteer_evaluation_script__:6:17
  › ExecutionContext._evaluateInternal (node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:217:19)
  › ExecutionContext.evaluate (node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:106:16)
  › file://test/browser.js:331:2
  › withPage (file://test/helpers/with-page.js:7:3)
```

in the browser tests.

Reply:
An educated guess... I think what's happening is that the browser is trying to load `/index.js` but there's no such route on the `server` in the tests. We didn't need a route prior to this PR because Puppeteer itself (on the Node.js side) was reading the file at `path` from the filesystem and injecting it inline as a string in a script tag.

Reply:
I managed to solve it. Really weird that the `import` call didn't throw though.